### ISSUE TITLE
Respect STATIC_ROOT and MEDIA_ROOT from settings.py

### ISSUE THREAD

**Issue by sHtev** [isotoma/django-json-settings#1]
If STATIC_ROOT and MEDIA_ROOT are defined in settings.py, django-json-settings should not override them and force their re-definition in a local json file. Currently parsed django settings should be checked as well as the just-loaded env vars.

**Comment by chris-nlnz** [comment]
I agree (2 years later). It is a terrible choice to introduce "magic" variables; someone might find their `STATIC_ROOT` point to a `/var/` somewhere even though it doesn't appear to be specified like that anywhere in their project. It could cost a developer a lot of time to figure out that it's your library that is messing with these settings quietly.
